hackathon_site/registration/models.py

```diff
@@ -117,27 +135,59 @@ class Application(models.Model):
 
     q1 = models.TextField(
```

Comment:
The `TextField` questions are for long-answer text, since they get rendered as textareas instead of normal text inputs. Make this

```python
how_many_hackathons = models.CharField(max_length=50, choices=HACKATHON_NUMBER_CHOICES, null=False)
```